docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -470,24 +470,69 @@ Collection versions use `Semantic Versioning <https://semver.org/>`_ for version
 
 .. _migrate_to_collection:
 
-Migrating Ansible content to a collection
-=========================================
+Migrating Ansible content to a different collection
+====================================================
+
+To migrate content from one collection to another, you need to create three PRs as follows:
+
+#. Create a PR against the old collection to remove the content.
+#. Create a PR against the new collection to add the files removed in step 1.
+#. Update the ``ansible/ansible:devel`` branch entries for all files moved.
+
+
+Removing the content from the old collection
+----------------------------------------------
+
+Create a PR against the old collection repo to remove the modules, module_utils, plugins, and docs_fragments related to this migration:
+
+#. If you are moving an action plugin, include the corresponding module with documentation.
+#. If you are moving a module, move any corresponding action plugin that should stay with it.
+#. Remove any entries about removed plugins from ``meta/action_groups.yml`` and ``meta/runtime.yml``. Ensure they are added into the new repo.
+#. Remove sanity ignore lines from ``tests/sanity/ignore\*.txt``
+#. Remove associated integration tests from ``tests/integrations/targets/`` and unit tests from ``tests/units/plugins/``.
+#. if you are moving from content from ``community.general`` or ``community.network``, remove entries from ``.github/BOTMETA.yml``.
```

Comment:
```suggestion
#. If you are moving from content from ``community.general`` or ``community.network``, remove entries from ``.github/BOTMETA.yml``.
```

Reply:
@felixfontein @gundalow - Alicia brought up some good points in discussing this with her.  This last four talk about removing items but aren't necessarily clear if you remove and forget, or if you are actually moving them, or redesigning them in the 2nd pr.  Can you help clarify this section?  for example:
 - ``meta/xx.yml`` - can you move them 'as is' to the new PR, or do you rewrite them somehow? 
- sanity ignore - I think these are clean removes. In the new PR, you may need to add similar (but not identical) entries into the new collection's ignore?
- tests - my guess is it's a rewrite in the new PR to account for FQCN etc?

Reply:
* `meta/runtime.yml` is tricky - it depends a lot on the content. Usually, new entries have to be added, and potentially some entries have to be removed (but probably not moved).
* sanity ignore: it's probably a move, but definitely a remove in the source collection. Once ansible-test can be configured per collection, it might not be a simple re-add in the target collection anymore, but right now it is.
* tests: it's usually copying them, and adjusting all FQCNs (same as in the plugins/modules themselves, their examples and references contain FQCNs that should potentially be updated).